tools/echo_server/echo_server.go

```diff
@@ -167,6 +215,11 @@ func logSetup() {
 }
 
 func main() {
+	serverPort := flag.String(portFlagName, "8883", "Port that will be used for this server.")
+	repeatMode := flag.Bool(repeatFlagName, false, "Flag indicating whether the echo server is in repeat mode.")
+	repeatIntervalSeconds := flag.Int(intervalFlagName, 1, "The repeat interval in seconds when in repeat mode.")
+	serverCert := flag.String(certFlagName, "", "X509 Server certificate for TLS connection. TLS will be enabled if this flag is set.")
```

Comment:
What is the reason for switching to specifying the certs and key values in the commandline? It seems more cumbersome than providing a file path. Particularly I am worried it may be hard to use on Windows (I personally tend to use zsh on MacOS and Unix).

Reply:
Because in this way it is easy for the IDT to run the echo server on an EC2 instance with a user-data script directly.
But I agree that this will be inconvenient for user who are using the config file. I will restore the certPath / keyPath options in next rev.

In next rev both file paths in config file and cmdline args can be used, with cmdline args having a higher priority.
i.e. (pseudo code)

```
if certFromCmdline exist {
  cert = loadCert(certFromCmdline)
} else {
  cert = loadCertFromPath(certPath)
}

```